sanity-io/structured-content-2022
# Update Sanity footer logo

# Update Sanity footer logo

## Intent

Update the Sanity logo in the footer to the newest version in Figma

## Description

Logo was exported from Figma; specifically, from the Desktop/Footer sketch.

Found it a bit tricky, but managed in the end by clicking one of the vector "segments", then clicking the target reticule for "parent component" in the Inspect tab (and then exporting to SVG via the Export tab).

## Testing this PR

1. Open the [front page](https://structured-content-2022-web-git-update-sanity-footer-logo.sanity.build/)
2. Scroll down to the bottom
3. Verify that the Sanity logo is there and is different from staging/matches the Desktop/Footer sketch in Figma. E.g. the lighter gray shades should have a bluish hue, and only the middle section of the S should be black
